app/scripts/controllers/meeting.ctrl.js

```diff
@@ -73,6 +73,10 @@ angular.module('dashboard')
         return (item && item.publicity) ? (item.publicity === PUBLICITY.SECRET) : false;
     };
 
+    self.isUrlString = function(url) {
+        return (url && (typeof url === "string") && url.length) ? true : false;
```

Comment:
The ternary operator `?:` is not really needed here but not really a problem, a matter of style.